packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/exporters/CompressionImageExporter.kt

```diff
@@ -2,68 +2,70 @@ package expo.modules.imagepicker.exporters
 
 import android.graphics.Bitmap
 import android.net.Uri
-import expo.modules.imagepicker.exporters.ImageExporter.Listener
-import expo.modules.interfaces.imageloader.ImageLoaderInterface
-import org.apache.commons.io.FilenameUtils
+import androidx.annotation.FloatRange
+import androidx.core.net.toFile
+import expo.modules.imagepicker.FailedToReadFileException
+import expo.modules.imagepicker.FailedToWriteFileException
+import expo.modules.imagepicker.toBitmapCompressFormat
+import expo.modules.kotlin.exception.ModuleNotFoundException
+import expo.modules.kotlin.providers.AppContextProvider
+import expo.modules.kotlin.providers.ContextProvider
+import kotlinx.coroutines.runInterruptible
 import java.io.ByteArrayOutputStream
 import java.io.File
+import java.io.FileNotFoundException
 import java.io.FileOutputStream
 import java.io.IOException
+import java.util.concurrent.ExecutionException
 
 class CompressionImageExporter(
-  private val mImageLoader: ImageLoaderInterface,
-  private val mQuality: Int,
-  private val mBase64: Boolean
+  private val appContextProvider: AppContextProvider,
+  private val contextProvider: ContextProvider,
+  @FloatRange(from = 0.0, to = 1.0)
```

Comment:
```suggestion
  @FloatRange(from = 0.0, to = 1.0)
```
Use the same validator from e-m-c